products/knownprojects/sql/aggregate/_aggregate_models.yml

```diff
@@ -32,5 +32,39 @@ models:
             - cdta
           config:
             severity: error
-            error_if: ">250"
+            error_if: ">{{ var('max_duplicate_projects') }}"
```

Comment:
when a condition for error is true, does it stop the pipeline?

Reply:
yup, but we can always set any or all tests to warn instead of error if we don't think they should stop a build